manifests/package/debian.pp

```diff
@@ -31,7 +31,7 @@
   }
 
   if $manage_repo {
-    include '::apt'
+    include 'apt'
```

Comment:
Stylewise... Do you prefer
```puppet
include 'apt'
```
or
```puppet
include apt
```

Reply:
I've no objections on this. In most cases I write it with the quotes.